recipe/deploy/update_code.php

```diff
@@ -110,6 +110,7 @@
     }
 
     // Save git revision in REVISION file.
-    $rev = escapeshellarg(run("$git rev-list $target -1"));
+    $targetRef = get('target');
+    $rev = escapeshellarg(run("$git rev-list $targetRef -1"));
     run("echo $rev > {{release_path}}/REVISION");
```

Comment:
Let's use {{target}} and rename traget to targetWithDir.

Reply:
Sounds good, I'll push the update in a moment.